src/components/organisms/pupil/OakQuizCheckBox/OakQuizCheckBox.tsx

```diff
@@ -204,7 +200,8 @@ export const OakQuizCheckBox = (props: OakQuizCheckBoxProps) => {
         isHighlighted ? "border-decorative5-stronger" : "transparent"
       }
     >
-      <StyledInternalCheckBoxLabelHoverDecor
+      <InternalCheckBoxLabelHoverDecor
```

Comment:
Is this change intentional ., I'm worried about regressions on the pupil journey.

Reply:
Ah got it. That makes sense.